include/pool/pool_disjoint.h

```diff
@@ -10,32 +10,45 @@ extern "C" {
 #include <umf/memory_pool.h>
 #include <umf/memory_provider.h>
 
-inline constexpr size_t MIN_BUCKET_DEFAULT_SIZE = 8;
+#define UMF_DISJOINT_POOL_MIN_BUCKET_DEFAULT_SIZE (8lu)
 
 struct umf_disjoint_pool_params {
     // Minimum allocation size that will be requested from the system.
     // By default this is the minimum allocation size of each memory type.
-    size_t SlabMinSize = 0;
+    size_t SlabMinSize;
 
     // Allocations up to this limit will be subject to chunking/pooling
-    size_t MaxPoolableSize = 0;
+    size_t MaxPoolableSize;
 
-    // When pooling, each bucket will hold a max of 4 unfreed slabs
-    size_t Capacity = 0;
+    // When pooling, each bucket will hold a max of 'Capacity' unfreed slabs
+    size_t Capacity;
 
     // Holds the minimum bucket size valid for allocation of a memory type.
     // This value must be a power of 2.
-    size_t MinBucketSize = MIN_BUCKET_DEFAULT_SIZE;
+    size_t MinBucketSize;
 
     // Holds size of the pool managed by the allocator.
-    size_t CurPoolSize = 0;
+    size_t CurPoolSize;
 
     // Whether to print pool usage statistics
-    int PoolTrace = 0;
+    int PoolTrace;
 };
 
 extern struct umf_memory_pool_ops_t UMF_DISJOINT_POOL_OPS;
 
+static inline struct umf_disjoint_pool_params umf_disjoint_pool_params_default() {
+    struct umf_disjoint_pool_params params = {
+        0,
```

Comment:
I know these parameters are defined above, but could you add comments here or assign defaults field by field?

Reply:
Done. Unfortunately, we cannot use designated initializers since they are not supported in C++ (until C++20)